cmd/tether/ops_linux.go

```diff
@@ -80,8 +82,20 @@ func (t *operations) SessionLog(session *tether.SessionConfig) (dio.DynamicMulti
 		log.Errorf("Setting terminal speed failed with %s", err)
 	}
 
```

Comment:
My previous comment got lost for some reason,  I think you should also set ttys1 to raw mode